.github/workflows/release.yml

```
@@ -0,0 +1,151 @@
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: "1.23"
+          cache-dependency-path: v2/go.sum
+
+      - name: Import GPG key (optional, for signing)
+        id: import_gpg
+        if: secrets.GPG_PRIVATE_KEY != ''
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)